pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.codice</groupId>
     <artifactId>usng4j</artifactId>
-    <version>0.2-SNAPSHOT</version>
+    <version>0.3-SNAPSHOT</version>
```

Comment:
Have we released 0.2?

Reply:
That's what's in current `master`, considering we are adding new functionality I thought we should increase the minor version; please let me know if my assumption was wrong and I need to change it back.

Reply:
It looks like only .1 has been released. I think this should remain at .2-snapshot

Reply:
If we are adding new functionality, it may warrant a release; we shouldn't bump development versions without releasing

Reply:
Done, updated it back to 0.2.